docs/current_docs/manuals/developer/snippets/documentation/typescript/object.ts

```diff
@@ -0,0 +1,24 @@
+import { object, field, func } from "@dagger.io/dagger"
+
+/**
+ * The object represents a single user of the system.
+ */
+@object()
+class MyModule {
+  /**
+   * The name of the user.
+   */
+  @field()
+  name: string
+
+  /**
+   * The age of the user.
+   */
+  @field()
+  age: number
+
+  constructor(age: number, name: string) {
```

Comment:
Constructor arguments here also not documented, that's why their flags don't show the descriptions.

Reply:
This also didn't work with the code below

```
import { object, field, func } from "@dagger.io/dagger"

/**
 * The object represents a single user of the system.
 */
@object()
class MyModule {
  /**
   * The name of the user.
   */
  @field()
  name: string

  /**
   * The age of the user.
   */
  @field()
  age: number

  constructor(age: number, name: string) {
    // The name of the user.
    this.name = name
    // The age of the user.
    this.age = age
  }
}
```
```
FUNCTIONS
  age           The age of the user.
  name          The name of the user.

ARGUMENTS
      --age int       [required]
      --name string   [required]
```

Reply:
You need to document the code following JsDoc convention with /** **/ comment

Reply:
Done